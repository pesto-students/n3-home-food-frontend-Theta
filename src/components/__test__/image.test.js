import * as React from "react";
import renderer from "react-test-renderer";
import Image from "image/image";

it("data not found", () => {
  const tree = renderer
    .create(<Image height={200} width={200} url="" redius="20px" />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
