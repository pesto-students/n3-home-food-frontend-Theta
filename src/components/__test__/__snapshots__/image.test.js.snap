// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`data not found 1`] = `
<img
  alt="not found"
  height={200}
  onError={[Function]}
  src=""
  style={
    Object {
      "backgroundPosition": "center",
      "borderRadius": undefined,
    }
  }
  width={200}
/>
`;
